OWML.Common/Menus/IModPauseMenu.cs

```diff
@@ -3,7 +3,7 @@
     public interface IModPauseMenu : IModPopupMenu
     {
         IModTabbedMenu OptionsMenu { get; }
-        
+
```

Comment:
Accidental empty line here?

Reply:
I run automated formatter through all files to make sure all tabulations are spaces,
pretty that was leftover spaces being deleted